lib/generators/atlassian_jwt_authentication/templates/jwt_tokens_migration.rb.erb

```diff
@@ -1,4 +1,4 @@
-class CreateAtlassianJwtTokens < ActiveRecord::Migration
+class CreateAtlassianJwtTokens < ActiveRecord::Migration[6.0]
```

Comment:
I don't mind Rails 6 here but we should probably use some older version, what do you think @bulinutza ?

Reply:
Unfortunately our projects are not on Rails 6 yet, so this wouldn't work for us. I'd suggest we use 5.